src/main/java/org/c4sg/controller/AllForGoodFeedController.java

```diff
@@ -0,0 +1,87 @@
+package org.c4sg.controller;
+
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import org.c4sg.dto.OrganizationDTO;
+import org.c4sg.dto.ProjectDTO;
+import org.c4sg.dto.afg.FeedDTO;
+import org.c4sg.dto.afg.FeedDateTimeDurationDTO;
+import org.c4sg.dto.afg.FeedInfoDTO;
+import org.c4sg.dto.afg.FeedLocationDTO;
+import org.c4sg.dto.afg.FeedSponsoringOrganizationDTO;
+import org.c4sg.service.OrganizationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+
+@CrossOrigin
+@RestController
+@RequestMapping("/api/allforgood")
+@Api(description = "AllforGood Feed Responses", tags = "feed")
+public class AllForGoodFeedController {
+	@Autowired
+    private OrganizationService organizationService;
+	
+	@CrossOrigin
+    @RequestMapping(method = RequestMethod.GET, produces = MediaType.APPLICATION_XML_VALUE)
```

Comment:
There is almost no documentation for returning JSON on the AllForFeed site. Here is the link tho their documentation: http://allforgoodorg.devcloud.acquia-sites.com/feed-specification